transformer/classes/dockerfile/rubydockerfilegenerator.go

```diff
@@ -0,0 +1,165 @@
+/*
+ *  Copyright IBM Corporation 2021
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package dockerfile
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/konveyor/move2kube/common"
+	"github.com/konveyor/move2kube/environment"
+	"github.com/konveyor/move2kube/types/qaengine/commonqa"
+	transformertypes "github.com/konveyor/move2kube/types/transformer"
+	"github.com/konveyor/move2kube/types/transformer/artifacts"
+	"github.com/sirupsen/logrus"
+)
+
+// RubyDockerfileGenerator implements the Transformer interface
+type RubyDockerfileGenerator struct {
+	Config       transformertypes.Transformer
+	Env          *environment.Environment
+	NodejsConfig NodejsDockerfileYamlConfig
```

Comment:
Shouldn't it be Ruby?

Reply:
Yes, it should be Ruby. I missed it during the find and replace. It’s my bad.